source/configure/manage-user-surveys.rst

```diff
@@ -0,0 +1,79 @@
+Manage user surveys
+===================
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
+  :start-after: :nosearch:
+
+Use the Mattermost User Survey integration to gather direct feedback from your Mattermost users to identify what's working well and what's not with your Mattermost instance. All user responses are stored in and remain within your self-hosted deployment. You can export a CSV report of NPS scores and user responses for further analysis or to share with Mattermost.
+
+You can schedule when each survey begins, define how long each survey lasts, specify teams excluded from the survey, as well as customize both a welcome message and a user question you want feedback on.
+
+Setup
+------
+
+You must be a Mattermost system admin to `upload the plugin <#upload>` to your Mattermost self-hosted deployment `enable it <#enable>`__,  `create surveys <#create-surveys>`__, and `export survey responses <#export-survey-responses>` using the System Console.
+
+The User Survey integration is compatible with the following :doc:`Mattermost Server versions </deploy/mattermost-changelog>`:
+
+  - v9.6 or later
```

Comment:
@harshilsharma63 - Is this server compatibility list correct for the user survey plugin?

Reply:
I believe we had stated v8.1 (the one that just came out of support) or later

Reply:
Sorry, just noticed the rest of the list below the comment

Reply:
@marianunez are we supporting v8.1? I don't think this has been tested on it. @yasserfaraazkhan can you confirm what versions did we test this on?

Reply:
@harshilsharma63 @marianunez 
Have verified the plugin works on 9.5.2, 9.8 and 9.9 server versions.

I can verify on other versions, if you suggest.

Reply:
Thanks, @yasserfaraazkhan. I've updated the docs to reflect the releases we've tested on. I'll defer to @marianunez and @harshilsharma63 on whether we want to test additional releases.